pallets/xcm-transactor/src/weights.rs

```diff
@@ -69,55 +73,95 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: XcmTransactor IndexToAccount (r:1 w:1)
 	#[rustfmt::skip]
 	fn register() -> Weight {
-		(19_401_000 as Weight)
+		(17_998_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	// Storage: XcmTransactor IndexToAccount (r:0 w:1)
 	#[rustfmt::skip]
 	fn deregister() -> Weight {
-		(16_181_000 as Weight)
+		(14_817_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	// Storage: XcmTransactor TransactInfoWithWeightLimit (r:0 w:1)
 	#[rustfmt::skip]
 	fn set_transact_info() -> Weight {
-		(19_595_000 as Weight)
+		(17_470_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	// Storage: XcmTransactor TransactInfoWithWeightLimit (r:0 w:1)
 	#[rustfmt::skip]
 	fn remove_transact_info() -> Weight {
-		(18_691_000 as Weight)
+		(16_694_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+	// Storage: XcmTransactor DestinationFeePerSecond (r:0 w:1)
+	#[rustfmt::skip]
+	fn set_fee_per_second() -> Weight {
+		(16_640_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: XcmTransactor TransactInfoWithWeightLimit (r:1 w:0)
+	// Storage: XcmTransactor DestinationFeePerSecond (r:1 w:0)
+	// Storage: ParachainInfo ParachainId (r:1 w:0)
+	// Storage: PolkadotXcm SupportedVersion (r:1 w:0)
+	// Storage: PolkadotXcm VersionDiscoveryQueue (r:1 w:1)
+	// Storage: PolkadotXcm SafeXcmVersion (r:1 w:0)
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
```

Comment:
We can probably whitelist some of these

Reply:
Yeah I though about that. If any, I think the parachainId one, since the others are the worst case scenario.